settings.gradle.kts

```diff
@@ -10,44 +10,51 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
+rootProject.name = "kermit-root"
 
-include(":kermit")
-include(":kermit-crashlytics")
-include(":kermit-crashlytics-test")
-include(":kermit-bugsnag")
-include(":kermit-bugsnag-test")
-include(":kermit-test")
-include(":kermit-koin")
-include(":kermit-sentry")
-include(":kermit-sentry-test")
+include(
+    ":kermit",
+    ":kermit-crashlytics",
+    ":kermit-crashlytics-test",
+    ":kermit-bugsnag",
+    ":kermit-bugsnag-test",
+    ":kermit-test",
+    ":kermit-koin",
+    ":kermit-sentry",
+    ":kermit-sentry-test",
 
-include(":kermit-gradle-plugin")
-include(":kermit-ir-plugin")
-include(":kermit-ir-plugin-native")
+    ":kermit-gradle-plugin",
+    ":kermit-ir-plugin",
+    ":kermit-ir-plugin-native",
 
-project(":kermit-gradle-plugin").projectDir = File("plugin/kermit-gradle-plugin")
-project(":kermit-ir-plugin").projectDir = File("plugin/kermit-ir-plugin")
-project(":kermit-ir-plugin-native").projectDir = File("plugin/kermit-ir-plugin-native")
```

Comment:
I guess this directory change was done so that the `allprojects{}` block wouldn't add config to the 'empty' `:plugin` project? The `allprojects{}` block has been removed (because the config will be applied using convention plugins), so I've changed the config here to be more wysiwyg.